### PR TITLE
Add *nix build during PR

### PR DESCRIPTION
**Pull request by ellismg**

### CODE REVIEW

**Comment by ellismg** [comment]
@dotnet-bot test CI please

**Comment by ellismg** [comment]
@dotnet-bot test CI Please

File: netci.groovy [hunk 1]
```diff
@@ -10,21 +10,28 @@ def buildString = '''call "C:\\Program Files (x86)\\Microsoft Visual Studio 14.0
 
 // Generate the builds for debug and release
 
-[true, false].each { isPR ->
-    def newJob = job(Utilities.getFullJobName(project, '', isPR)) {
-      steps {
-        batchFile(buildString)
```
Comment (mellinoe) [inline netci.groovy]: nit: I guess `buildString` is unused now.

Comment (ellismg) [inline netci.groovy]: Ahh, will remove.

**Comment by ellismg** [comment]
@dotnet-bot test CI please

**Pull request merged**